### ISSUE TITLE
The details about loss?

### ISSUE THREAD

**Issue by Fanke-max** [zpbao/Discovery_Obj_Move#1]
Hello! I am very interested in this job, which is the first to be able to make the slot attention work on video. In our attempt to implement DOM, we face some problems. In the paper, the authors add all the MSE and cross-entropy loss together. However, it seems such a strategy will make the training unstable. Could you please tell us how you calculate the loss? Did you take the average among the time or add them together along the batch or time dimension? And should we take the average on the spatial dimension when calculating MSE and cross-entropy loss？ What's more, it seems the paper did not specify what matrix norm in the slot consistency. I take the sum of absolute number, is it okay？

**Comment by Fanke-max** [comment]
And what did you take to calculate the matching loss of cross-entropy between mask and prediction？Sum or average？

**Comment by zpbao** [comment]
Hi Fanke! Thanks for your interest and sorry for the late reply! I uploaded our training code and I think all (or maybe most) of the issues can be found in the file `train.py.` Feel free to make further discussions, thanks!
